pom.xml

```diff
@@ -44,7 +44,7 @@
         <air.maven.version>3.3.9</air.maven.version>
 
         <dep.antlr.version>4.7.1</dep.antlr.version>
-        <dep.airlift.version>0.183</dep.airlift.version>
+        <dep.airlift.version>0.183.5</dep.airlift.version>
```

Comment:
`0.183.5` is managed here.
https://github.com/treasure-data/airlift/commits/0.183.1-for-td